packages/xo-server/package.json

```diff
@@ -86,8 +88,8 @@
     "iterable-backoff": "^0.1.0",
     "jest-worker": "^26.3.0",
     "js-yaml": "^3.10.0",
-    "json-stringify-safe": "^5.0.1",
     "json-rpc-peer": "^0.17.0",
+    "json-stringify-safe": "^5.0.1",
```

Comment:
You can rebase, I've sorted on `master` to keep this PR clean :slightly_smiling_face: